---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: 'SimPrily: A Python framework to simplify high-throughput genomic simulations'
subtitle: ''
summary: ''
authors:
- Ariella L. Gladstein
- Consuelo D. Quinto-Cort√©s
- Julian L. Pistorius
- David Christy
- Logan Gantner
- Blake L. Joyce
tags: []
categories: []
date: '2018-01-01'
lastmod: 2020-09-04T17:25:38-04:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-09-04T21:25:38.162477Z'
publication_types:
- 2
abstract: Genomic simulations are an important technique used in population genetics
  to infer demographic history, test for regions under selection, and create datasets
  to validate software. However, running thousands of simulations and manipulating
  large loci can present computational challenges. We present SimPrily, a Python tool
  optimized for high throughput computing (HTC), which facilitates simulation of whole
  chromosomes. SimPrily can use prior distributions of parameters to run simulations,
  incorporate single nucleotide polymorphism array ascertainment bias into the simulated
  model, and calculate a variety of genomic summary statistics. We include with SimPrily
  high-throughput workflows that leverage free computing resources through the Open
  Science Grid and CyVerse Discovery Environment, allowing researchers to run thousands
  or millions of large-locus simulations with minimal or no prior command line knowledge.
publication: '*SoftwareX*'
url_pdf: https://www.sciencedirect.com/science/article/pii/S2352711018301213
doi: 10.1016/J.SOFTX.2018.09.003
---
